vita3k/app/src/app_init.cpp

```diff
@@ -103,83 +103,116 @@ void update_viewport(EmuEnvState &state) {
 }
 
 void init_paths(Root &root_paths) {
-    const auto dir_sep = std::string{ fs::path::preferred_separator };
-    auto base_path = SDL_GetBasePath();
-    auto pref_path = SDL_GetPrefPath(org_name, app_name);
-    root_paths.set_base_path(string_utils::utf_to_wide(base_path));
-    root_paths.set_pref_path(string_utils::utf_to_wide(pref_path));
-    root_paths.set_log_path(string_utils::utf_to_wide(base_path));
-    root_paths.set_config_path(string_utils::utf_to_wide(base_path));
-    root_paths.set_static_assets_path(string_utils::utf_to_wide(base_path));
-    root_paths.set_shared_path(string_utils::utf_to_wide(base_path));
-    root_paths.set_cache_path(fs::path(string_utils::utf_to_wide(base_path)) / "cache" / dir_sep);
-    SDL_free(base_path);
-    SDL_free(pref_path);
+    auto base_path_sdl = SDL_GetBasePath();
+    fs::path base_path(base_path_sdl);
```

Comment:
May be incorrect for windows earlier then Version 1903 (May 2019 Update) (see here https://learn.microsoft.com/en-us/windows/apps/design/globalizing/use-utf8-code-page). SDL uses utf8 everywhere, so we need to convert from utf8 to wstring (or current application codepage) directly.

Reply:
Should be fixed, thanks.